js/freewheel.js

```diff
@@ -671,6 +672,14 @@ OO.Ads.manager(function(_, $) {
       }
     };
 
+    var onRePlayRequested = function() {
+      _resetAdState();
+      shouldRequestAds = true;
```

Comment:
@pilievOoyala Change made - created a new function and had the two other functions call it.